test/hmc_core.jl/bayes_lr.jl

```diff
@@ -81,9 +81,10 @@ lj = lj_func(θ)
 chn = []
 accept_num = 1
 
-
+#=
```

Comment:
Maybe add a note as a reminder that this should be reverted?

Reply:
Sure thing.